src/app/pages/onboarding/set-password/set-password.tsx

```diff
@@ -108,7 +108,7 @@ export const SetPasswordPage = () => {
       .defined()
       .when('password', {
         is: (val: string | any[]) => (val && val.length > 0 ? true : false),
-        then: yup.string().oneOf([yup.ref('password')], 'Passwords must match'),
+        then: yup.string().oneOf([yup.ref('password'), null], 'Passwords must match'),
```

Comment:
What does adding the `null` value here, do?

Reply:
This include null case, but it maybe not useful in our case